opslevel/resource_opslevel_trigger_definition.go

```diff
@@ -101,7 +101,11 @@ func (r *TriggerDefinitionResource) Schema(ctx context.Context, req resource.Sch
 					"The entity type to associate with the Trigger Definition. One of `%s`",
 					strings.Join(opslevel.AllCustomActionsEntityTypeEnum, "`, `"),
 				),
+				Computed: true,
 				Optional: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 				Validators: []validator.String{
 					stringvalidator.OneOf(opslevel.AllCustomActionsEntityTypeEnum...),
 				},
```

Comment:
We have other optional fields here? (and probably more across the provider)  Does this change need to be applied to them too?  If so we should probably cut a ticket to audit.)

Reply:
I had the same thought and looked around. This field is unlike most (not all) fields in the way that the API provides a default value if no value is given. [Follow up ticket created](https://github.com/OpsLevel/team-platform/issues/378)